.github/workflows/ci.yaml

```diff
@@ -88,3 +88,41 @@ jobs:
           env_vars: OS,PYTHON
           fail_ci_if_error: True
           verbose: True
+
+  check_packaging:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: mamba-org/setup-micromamba@v1
+      with:
+          environment-file: environment.yml
+          environment-name: kartograf
+
+    - name: install_deps
+      run: |
+        python -m pip install -U pip pipx wheel build
+
+    - name: build_package_sdist
+      run: pipx run build --sdist
+
+    - name: check_package_build
+      run: |
+        DISTRIBUTION=$(ls -t1 dist/kartograf-*.tar.gz | head -n1)
+        test -n "${DISTRIBUTION}" || { echo "no distribution dist/kartograf-*.tar.gz found"; exit 1; }
+
+    - name: install sdist
+      working-directory: ./dist
+      run: |
+        ls -a .
+        python -m pip install kartograf-*.tar.gz
+        du -sh kartograf-*.tar.gz
+
+    - name: check install
+      run: pip list
+
+    - name: run tests
+      working-directory: ./dist
```

Comment:
This was me mostly being paranoid that I'd pick up the actual package directory when executing this at the top of the repo. In a `src` layout repo, this risk probably goes away, so it probably is harmless to remove this.

Reply:
I prefer doing it this way to be sure!